src/Kovan-DssSpell.sol

```diff
@@ -74,6 +72,7 @@ contract SpellAction {
     // A table of rates can be found at
     //    https://ipfs.io/ipfs/QmefQMseb3AiTapiAKKexdKHig8wroKuZbmLtPLv4u2YwW
     //
+    uint256 constant ONE_PERCENT_RATE   = 1000000000315522921573372069;
```

Comment:
Any reason we're not pulling these from rates.sol?

Reply:
We can only do this in the tests. To do it in the spell it would deploy the whole Rates contract.

Reply:
Oh duh I missed that this was the actual spell.